beta/src/content/learn/referencing-values-with-refs.md

```diff
@@ -641,7 +639,7 @@ export default function Chat() {
   return (
     <>
       <input
-        value={text}
+        defaultValue={textRef.current}
```

Comment:
The problem here is that this example reads from a ref during render. Even if that specific prop is only relevant once, the overall pattern is still problematic. React might even warn if `defaultValue` changes. But the biggest problem is reading from a ref during render which we don't want to encourage.